src/JSON For mIRC.mrc

```diff
@@ -1639,12 +1639,12 @@ alias -l jfm_Create {
   jfm_log -I $!jfm_create( $+ $1 $+ , $+ $2 $+ , $+ $3 $+ , $+ $4)
 
   ;; Attempt to create the json handler and if an error occurs retrieve the error, log it and return it
-  if (!$com(SReject/JSONForMirc/JSONEngine, JSONCreate, 1, bstr, $2, &bstr, $3, bool, %NoParse, dispatch* $1) || $comerr || !$com($1)) {
+  if (!$com(SReject/JSONForMirc/JSONEngine, JSONCreate, 1, bstr, $2, &bstr, $3, bool, %NoParse, dispatch* $1)) || ($comerr) || (!$com($1)) {
     %Error = $jfm_GetError
   }
 
   ;; Attempt to call the parse method if the handler should not wait for the http request
-  elseif ($2 !== http || ($2 == http && !%Wait)) && (!$com($1, parse, 1)) {
+  elseif ($2 !== http) || ($2 == http) && (!%Wait) && (!$com($1, parse, 1)) {
```

Comment:
This is incorrect. The correct version would be the following but I choose not to include the extra `()`:

```
elseif (($2 !== http) || (($2 == http) && (!%Wait))) && (!$com($1, parse, 1)) {
```